python_modules/dagster/dagster/core/definitions/repository.py

```diff
@@ -381,10 +403,10 @@ def has_sensor(self, sensor_name):
 class CachingRepositoryData(RepositoryData):
     """Default implementation of RepositoryData used by the :py:func:`@repository <repository>` decorator."""
 
-    def __init__(self, pipelines, partition_sets, schedules, sensors):
+    def __init__(self, pipelines_and_jobs, partition_sets, schedules, sensors):
```

Comment:
With my original comment on combining pipelines and jobs together, the intention was to still have distinct parameters for each in this function (and to have the cache continue to store them separately). The idea was more about the logical organization in the from_list function (where you have the benefit of knowing if an individual item is a JobDefinition or a PipelineDefinition, unlike when things are already in a dictionary).

Reply:
Ahhhh ok this makes more sense. Reverted back to the original and changed the `from_list` function